fastlane/lib/fastlane/actions/commit_version_bump.rb

```diff
@@ -61,8 +78,11 @@ def self.run(params)
           end
         end
 
+        extra_files = params[:include]
+        extra_files += modified_files_relative_to_repo_root repo_path
```

Comment:
Ah! sorry I missed this before @jdee - can you just wrap this parameter in parens?